pkg/network/transport/object/grpc/service.go

```diff
@@ -24,6 +24,11 @@ func New(c objectSvc.ServiceServer) *Server {
 	}
 }
 
+// SetSrv set service server.
+func (s *Server) SetSrv(c objectSvc.ServiceServer) {
```

Comment:
Added because we need to exclude/include metrics request handler from chain. When metrics disabled I set for object service as next service in chain sign service, when enabled - metrics. Agree that it is better to disable calculation inside metrics service by flag.